Makefile

```diff
@@ -117,6 +117,7 @@ ci:	## Run in automation environment
 
 .PHONY: terraform-app-init
 terraform-app-init:
+		$(if $(tag), , $(eval export tag=master))
```

Comment:
Can this be moved to a separate commit to emphasize the new behaviour? And this would be helpful to document it.